src/cli.js

```diff
@@ -10,14 +10,14 @@ const exec = require('child_process').execSync
 const dedent = require('dedent-js')
 const getSymlinkedDependencies = require('./getSymlinkedDependencies')
 const getMetroConfig = require('./getMetroConfig')
-const getDependencyPath = require('./getDependencyPath')
 
 const CONFIG_FILENAME = 'rn-cli.config.js'
 
-const mapDep = dep => `    - ${dep} -> ${getDependencyPath(dep)}`
+const mapDep = dep => `    - ${dep}`
```

Comment:
💁 Full paths are now being passed into this, so it doesn't make sense to have package names and then mapped paths anymore. Could use cleanup.